lib/v2/zooTeam/radar.js

```diff
@@ -0,0 +1,24 @@
+module.exports = {
+    'weekly.zoo.team': {
+        _name: '政采云前端小报',
+        '.': [
+            {
+                title: '首页',
+                docs: 'https://docs.rsshub.app/blog.html#zheng-cai-yun-qian-duan-ji-shu-tuan-dui-xiao-bao',
+                source: '/',
+                target: '/zooTeam/weekly',
+            },
+        ],
+    },
+    'zoo.team': {
+        _name: '政采云前端博客',
+        '.': [
+            {
+                title: '首页',
+                docs: 'https://docs.rsshub.app/blog.html#zheng-cai-yun-qian-duan-ji-shu-tuan-dui-bo-ke',
+                source: '/',
+                target: '/zooTeam/blog',
+            },
+        ],
+    },
```

Comment:
```suggestion
    'zoo.team': {
        _name: '政采云前端技术团队',
        weekly: [
            {
                title: '小报',
                docs: 'https://docs.rsshub.app/blog.html#zheng-cai-yun-qian-duan-ji-shu-tuan-dui',
                source: '/',
                target: '/zooTeam/weekly',
            },
        ],
        '.': [
            {
                title: '博客',
                docs: 'https://docs.rsshub.app/blog.html#zheng-cai-yun-qian-duan-ji-shu-tuan-dui',
                source: '/',
                target: '/zooTeam/blog',
            },
        ],
    },
```
`_name` should match H2 header of the docs, `title` should match H3 header of the docs.